source/game/item/ItemInventory.cc

```diff
@@ -21,6 +21,11 @@ void ItemInventory::useItem(int count) {
         return;
     }
 
+    clear();
+}
+
+/// @addr{0x807BC9F4}
```

Comment:
```suggestion
/// @addr{0x807BC9C0}
```
This is a tricky one.

This is an inline that exists in both `useItem` and the function responsible for deleting items on player respawn. We aren't dealing with clearing any actual world objects, which is the only difference between this function and the player respawn one.